front-end/src/store/interfaces/auth.interfaces.ts

```diff
@@ -29,8 +29,16 @@ export interface UserCreds {
 
 export type TempAuthUSer = Partial<UserCreds>;
 
+export enum Roles {
+	USER = 1,
+	PRODUCT_OWNER = 2,
+	PEDAGOGUE = 3,
+}
+
 export interface User {
-	role?: string;
+	id?: string;
+	_id?: string;
+	role?: Roles;
 	profile?: UserProfile;
 	schoolProfile?: UserSchoolProfile;
 	businessProfile?: UserBusinessProfile;
```

Comment:
Need an explanation about the myArticles and events, they shouldn't be on the user itself.